README.adoc

```diff
@@ -102,11 +145,33 @@ cd /home/project/guide-jakarta-faces/start
 endif::[]
 
 When you run Open Liberty in https://openliberty.io/docs/latest/development-mode.html[dev mode^], dev mode listens for file changes and automatically recompiles and deploys your updates whenever you save a new change. Run the following goal to start Open Liberty in dev mode:
+include::{common-includes}/os-tabs.adoc[]
+
+[.tab_content.windows_section]
+--
+[role='command']
+```
+mvnw.cmd liberty:dev
+```
+--
+
+[.tab_content.mac_section]
+--
+[role='command']
+```
+cd start
```

Comment:
Why does Mac and Linux have `cd start` but Windows doesn't? I would remove it since the previous paragraph says to "Navigate to the start directory".